codersdk/agentconn.go

```diff
@@ -18,23 +18,35 @@ import (
 	"github.com/coder/coder/tailnet"
 )
 
+var (
+	// TailnetIP is a static IPv6 address with the Tailscale prefix that is used to route
+	// connections from clients to this node. A dynamic address is not required because a Tailnet
+	// client only dials a single agent at a time.
+	TailnetIP                  = netip.MustParseAddr("fd7a:115c:a1e0:49d6:b259:b7ac:b1b2:48f4")
+	TailnetSSHPort             = 1
+	TailnetReconnectingPTYPort = 2
+	TailnetSpeedtestPort       = 3
```

Comment:
Why move these?

Reply:
They aren't useful to anyone consuming the SDK which is why I ask.

Reply:
Yup, that makes sense! I think we can make this private.

Reply:
The agent package imports these now, as well as the codersdk package.